tests/Unit/Domain/Test_BlockAndElementLogicalCoordinates.cpp

```diff
@@ -714,7 +715,7 @@ void test_block_logical_coordinates_with_roundoff_error() {
   std::vector<std::array<double, 3>> points{};
   std::vector<size_t> expected_block_ids{};
   // Block piercing z has ID 0, block piercing x has ID 4. Phi=0, Theta=Pi/4 is
-  // on their shared boundary. See also WedgeOrientations.png in the Shell docs.
+  // on their shared boundary. See also WedgeOrientations.png in the Wedge docs.
```

Comment:
`Wedge` -> `Sphere` (the Wedge docs don't have this image, but you added the png to the Sphere documentation :relaxed:  )

also on line 758